lib/api/listParts.js

```diff
@@ -177,6 +178,8 @@ function listParts(authInfo, request, log, callback) {
             return services.getSomeMPUparts(getPartsParams,
             (err, storedParts) => {
                 if (err) {
+                    monitoring.promMetrics('GET', bucketName, err.code,
```

Comment:
as discussed with you, you only catch this error path, not the previous one (line 147 for example). You can catch all of them at line 274, same for the success path you catch at line 269, it can be done line 274

Reply:
it is exactly what you do for  multiObjectDelete ;)

Reply:
Yeah you are right, I will make the change